### PR TITLE
Creation of Size Function

### PR DESCRIPTION
**Pull request by zialodhi19**

### CODE REVIEW

**Comment by zialodhi19** [comment]
What should be done to avoid these Fails,
Kindly help

**Comment by zialodhi19** [comment]
Why is not anyone responding ?

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @zialodhi19 

Apologies for the late reply. Was swamped with work last week and so reviewing this PR totally slipped under my radar 😅 

With regards to your PR, could you kindly mention the issue# for the subtask this PR refers to?  From the file you've modified, it seems that the task is related to the JAX frontends but I don't see any tests written for it. Could you kindly write the tests for `jnp.size`
Let me know if you have any questions.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
